benches/bench.rs

```diff
@@ -40,7 +39,7 @@ fn _bench_enc<T: Abomonation>(bencher: &mut Bencher, record: T) {
     bencher.bytes = bytes.len() as u64;
     bencher.iter(|| {
         bytes.clear();
-        unsafe { encode(&record, &mut bytes).unwrap(); }
+        unsafe { encode(&record, &mut bytes).unwrap() }
```

Comment:
Scarily enough, a semicolon is all it takes to defeat Bencher's hidden `black_box` mechanism.